py3status/modules/bluetooth.py

```diff
@@ -55,47 +52,28 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     device_separator = '|'
-    format = '{name}'
-    format_no_conn = 'OFF'
-    format_no_conn_prefix = 'BT: '
-    format_prefix = 'BT: '
+    format = 'BT[: {format_device}]'
```

Comment:
> `{name}` or `{mac}` in format

yeah this is tricky.  I think maybe we need some deprecation code changes to do this all properly.

`format = 'BT: [{format_device}|\?show OFF]'`  I think is pretty close to the existing output and not too ugly